### PR TITLE
New styling for when a level is assessed and selected

### PR DESCRIPTION
**Pull request by alogan519**
Level that is assessed will now be displayed immediately

### CODE REVIEW

File: d2l-rubric-levels-mobile.html [hunk 1]
```diff
@@ -55,6 +62,12 @@
 				z-index: 0;
 				border: solid 1px rgba(0, 111, 191, 0.4);
 			}
+			.level.selected.assessed:focus {
```
Comment (walinga) [inline d2l-rubric-levels-mobile.html]: Here you have levels.selected.assessed but above you have levels.assessed.selected. Why not keep a consistent order?

File: d2l-rubric-levels-mobile.html [hunk 2]
```diff
@@ -120,7 +136,7 @@
 						aria-controls$="level-description-panel[[index]]"
 					>
 						<d2l-icon hidden$="[[!_isAssessedLevel(item, assessedLevelHref)]]"class="check-icon" icon='d2l-tier1:check'></d2l-icon>
-						<div hidden$="[[_isAssessedLevel(item, assessedLevelHref)]]" class$="[[_getLevelTextClassName(index, selected)]]">
+						<div hidden$="[[_isAssessedLevel(item, assessedLevelHref)]]" class$="[[_getLevelTextClassName(index, selected, item, assessedLevelHref)]]">
```
Comment (walinga) [inline d2l-rubric-levels-mobile.html]: You added parameters here, but you didn't update the function header

Comment (alogan519) [reply d2l-rubric-levels-mobile.html]: Looks line 247 on my diff

Comment (walinga) [reply d2l-rubric-levels-mobile.html]: _getLevelClassName vs _getLevelTextClassName. It's not the same function

**Pull request merged**